gr-qtgui/include/gnuradio/qtgui/freq_sink_c.h

```diff
@@ -83,7 +83,11 @@ class QTGUI_API freq_sink_c : virtual public sync_block
      * \param fftsize size of the FFT to compute and display. If using
      *        the PDU message port to plot samples, the length of
      *        each PDU must be a multiple of the FFT size.
-     * \param wintype type of window to apply (see gr::fft::window::win_type)
+     * \param wintype type of window to apply (see gr::fft::window::win_type).
+     *        By setting bit 16 to one, this block will normalize the window
```

Comment:
I wonder whether this hack is of any use. You can call `set_fft_window_normalized()` after calling the constructor (this is also what the template for GRC does).